tests/data/mock-raspa-e968334-1990dc391d097b6bec4877654b109877/_aiidasubmit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+exec > _scheduler-stdout.txt
+exec 2> _scheduler-stderr.txt
+
+
+
+'/Users/leopold/Personal/Programmierung/qm-codes/RASPA2/src/.libs/simulate'  'simulation.input'   
```

Comment:
Is this path correct?

Reply:
yes, it's the path on my PC. This is the "output directory" used by the parser, i.e. the content of _aiidasubmit.sh does not matter.

Reply:
ok